### ISSUE TITLE
fomantic 2.75 compined with meteor-up fails and is missing "lodash._basetostring"

### ISSUE THREAD

**Issue by LarsBuur** [fomantic/Fomantic-UI#782]
# Bug Report

I upgraded my semantic-ui package from 2.4.2 to fomantic-ui 2.7.5

Afterwards, I am unable to deploy my meteor application.

The change from semantic-ui 2.4.2 to fomantic-ui 2.7.5 is the only change.

I have tried to add lodash as an dependency to fomantic-ui with no change.

## Steps to reproduce
Let me know if this is non-trivial and I will try to create a repo with the bare minimum´s
1. npm install --save fomantic-ui
2. mup deploy

## Actual result
```
Building App Bundle Locally

Started TaskList: Pushing Meteor App
[5.103.25.140] - Pushing Meteor App Bundle to the Server
[5.103.25.140] - Pushing Meteor App Bundle to the Server: SUCCESS
[5.103.25.140] - Prepare Bundle
[5.103.25.140] x Prepare Bundle: FAILED

	      ------------------------------------STDERR------------------------------------
	      The command '/bin/sh -c cd  /built_app/programs/server &&     npm install --unsafe-perm' returned a non-zero code: 1

	      ------------------------------------STDOUT------------------------------------
	      odules/meteor/babel-compiler/node_modules/babel-plugin-transform-property-literals
	babel-plugin-transform-regexp-constructors@0.4.3 /built_app/programs/server/npm/node_modules/meteor/babel-compiler/node_modules/babel-plugin-transform-regexp-constructors
	babel-plugin-transform-remove-console@6.9.4 /built_app/programs/server/npm/node_modules/meteor/babel-compiler/node_modules/babel-plugin-transform-remove-console
	babel-plugin-transform-remove-debugger@6.9.4 /built_app/programs/server/npm/node_modules/meteor/babel-compiler/node_modules/babel-plugin-transform-remove-debugger
	babel-plugin-transform-remove-undefined@0.5.0 /built_app/programs/server/npm/node_modules/meteor/babel-compiler/node_modules/babel-plugin-transform-remove-undefined
	babel-plugin-transform-simplify-comparison-operators@6.9.4 /built_app/programs/server/npm/node_modules/meteor/babel-compiler/node_modules/babel-plugin-transform-simplify-comparison-operators
	babel-plugin-transform-undefined-to-void@6.9.4 /built_app/programs/server/npm/node_modules/meteor/babel-compiler/node_modules/babel-plugin-transform-undefined-to-void
	babel-preset-meteor@7.3.4 /built_app/programs/server/npm/node_modules/meteor/babel-compiler/node_modules/babel-preset-meteor
	babel-preset-minify@0.5.0 /built_app/programs/server/npm/node_modules/meteor/babel-compiler/node_modules/babel-preset-minify
	lodash.isplainobject@4.0.6 /built_app/programs/server/npm/node_modules/meteor/babel-compiler/node_modules/lodash.isplainobject
	json5@2.1.0 /built_app/programs/server/npm/node_modules/meteor/babel-compiler/node_modules/json5
	meteor-babel@7.3.4 /built_app/programs/server/npm/node_modules/meteor/babel-compiler/node_modules/meteor-babel
	meteor-babel-helpers@0.0.3 /built_app/programs/server/npm/node_modules/meteor/babel-compiler/node_modules/meteor-babel-helpers
	reify@0.17.3 /built_app/programs/server/npm/node_modules/meteor/babel-compiler/node_modules/reify

	> core-js@2.6.8 postinstall /built_app/programs/server/npm/node_modules/core-js
	> node -e "try { require('./scripts/postinstall'); } catch (e) { /* empty */ }"

	Thank you for using core-js ( https://github.com/zloirock/core-js )!

	Please consider supporting of core-js on Open Collective or Patreon:
	> https://opencollective.com/core-js
	> https://www.patreon.com/zloirock

	Also, the author of core-js ( https://github.com/zloirock ) is looking for a good job -)


	> fomantic-ui@2.7.5 install /built_app/programs/server/npm/node_modules/fomantic-ui
	> gulp install

	module.js:491
	    throw err;
	    ^

	Error: Cannot find module 'lodash._basetostring'
	    at Function.Module._resolveFilename (module.js:489:15)
	    at Function.Module._load (module.js:439:25)
	    at Module.require (module.js:517:17)
	    at require (internal/module.js:11:18)
	    at Object.<anonymous> (/built_app/programs/server/npm/node_modules/lodash.template/index.js:10:20)
	    at Module._compile (module.js:573:30)
	    at Object.Module._extensions..js (module.js:584:10)
	    at Module.load (module.js:507:32)
	    at tryModuleLoad (module.js:470:12)
	    at Function.Module._load (module.js:462:3)
	npm ERR! code ELIFECYCLE
	npm ERR! errno 1
	npm ERR! fomantic-ui@2.7.5 install: `gulp install`
	npm ERR! Exit status 1
	npm ERR!
	npm ERR! Failed at the fomantic-ui@2.7.5 install script.
	npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

	npm ERR! A complete log of this run can be found in:
	npm ERR!     /root/.npm/_logs/2019-05-29T10_06_41_073Z-debug.log
	npm ERR! code ELIFECYCLE
	npm ERR! errno 1
	npm ERR! meteor-dev-bundle@ install: `node npm-rebuild.js`
	npm ERR! Exit status 1
	npm ERR!
	npm ERR! Failed at the meteor-dev-bundle@ install script.
	npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

	npm ERR! A complete log of this run can be found in:
	npm ERR!     /root/.npm/_logs/2019-05-29T10_06_41_112Z-debug.log

	      ------------------------------------------------------------------------------
```
## Screenshot (when possible)
<img width="1401" alt="Skærmbillede 2019-05-29 kl  12 17 37" src="https://user-images.githubusercontent.com/3585707/58549438-caca0880-820b-11e9-91b4-0efe3db88469.png">

## Version
2.7.5

**Comment by hammy2899** [comment]
@LarsBuur What version of node and gulp are you using?

```
$ node -v
$ gulp -v
$ npx gulp -v
```

also have you tried deleting your node_modules and lock file and reinstalling your dependencies?

**Comment by LarsBuur** [comment]
Thank you so much for replying - it might just be an dependency issue. What made me write here is that it did work with the sui 2.4.2

node -v
v9.0.0

gulp -v
[14:07:25] CLI version 2.0.1
[14:07:25] Local version 4.0.2

npx gulp -v
CLI version: 2.2.0
Local version: 4.0.2

**Comment by LarsBuur** [comment]
att. @hammy2899  :-)

Thank you so much for replying - it might just be an dependency issue. What made me write here is that it did work with the sui 2.4.2

node -v
v9.0.0

gulp -v
[14:07:25] CLI version 2.0.1
[14:07:25] Local version 4.0.2

npx gulp -v
CLI version: 2.2.0
Local version: 4.0.2

**Comment by hammy2899** [comment]
@LarsBuur Sorry for the late reply, have you tried deleting your node_modules and lock file and reinstalling your dependencies? If that doesn't work try and update to the latest Node version.

**Comment by LarsBuur** [comment]
@hammy2899 - again thanks for taking the time. Meteor versions are pinned to specific node versions.  Currently node v8.15.1

I will try to clean out the node_modules and lock file and report back. I am on a ferry with poor internet atm.

**Comment by LarsBuur** [comment]
So I managed to fix this issue and for future references, my error was that I did 
```
npm install --save fomantic-ui
```
 which did install fomantic-ui and triggered the setup process, but messes up the meteor build system / meteor-up. 

The solution was to just remove fomantic-ui from package.json and revert the lock file thus running 
```
npm install fomantic-ui
```
fixed my problem.

Looking forward to test, use and contribute to fomantic-ui. Looking great!
